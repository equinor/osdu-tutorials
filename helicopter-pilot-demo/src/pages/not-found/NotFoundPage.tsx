import * as React from "react";

export function NotFoundPage() {
    return (
        <div className="not-found">
            <h2>Sorry. The page you are looking for was not found...</h2>
        </div>
    )
}