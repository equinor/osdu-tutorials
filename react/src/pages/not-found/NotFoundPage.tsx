import React from 'react';
import './styles.css';

/**
 * Show an empty page when url has not found
 */
export function NotFoundPage() {
  return (
    <div className="not-found">
      <h2>Sorry. The page you are looking for was not found...</h2>
    </div>
  );
}
