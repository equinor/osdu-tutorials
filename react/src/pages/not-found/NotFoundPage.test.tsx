import React from 'react';
import { shallow } from 'enzyme';
import { NotFoundPage } from './NotFoundPage';

describe('NotFoundPage', () => {
  it('should render self and subcomponents', () => {
    const wrapper = shallow(<NotFoundPage />);

    expect(wrapper).toMatchSnapshot();
  });
});
