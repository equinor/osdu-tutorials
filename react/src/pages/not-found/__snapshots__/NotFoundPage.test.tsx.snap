// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFoundPage should render self and subcomponents 1`] = `
<div
  className="not-found"
>
  <h2>
    Sorry. The page you are looking for was not found...
  </h2>
</div>
`;
